flink-doris-connector/src/main/java/org/apache/doris/flink/sink/schema/SchemaChangeHelper.java

```diff
@@ -44,6 +44,7 @@ public class SchemaChangeHelper {
     private static final String CREATE_DATABASE_DDL = "CREATE DATABASE IF NOT EXISTS %s";
     private static final String MODIFY_TYPE_DDL = "ALTER TABLE %s MODIFY COLUMN %s %s";
     private static final String MODIFY_COMMENT_DDL = "ALTER TABLE %s MODIFY COLUMN %s COMMENT '%s'";
+    private static final String SHOW_FULL_COLUMN_DDL = "SHOW FULL COLUMNS FROM `%s`.`%s`";
```

Comment:
Is this approach not universal?
This sql seems to be the unique syntax of MySQL. If the upstream is Oracle or PostgreSQL, it will not be supported.

Reply:
It only executes within Doris.